### PR TITLE
Bump [v114] Update Sentry to version 8.6.0

### PR DESCRIPTION
**Pull request by rvandermeulen**
Changelogs since previous update:
* https://github.com/getsentry/sentry-cocoa/releases/tag/8.5.0
* https://github.com/getsentry/sentry-cocoa/releases/tag/8.6.0

### CODE REVIEW

**Review by nishant2718 — APPROVED** [review]

**Comment by nishant2718** [comment]
These changes look reasonable to me, but I'd wait on @lmarceau's approval as well.

**Comment by lmarceau** [comment]
LGTM as well!

**Review by lmarceau — APPROVED** [review]

**Pull request merged**
